gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
+      # Alias with hardcoded :all method, just for convenience, since :all method will be used in 99% of the cases
+      handled_list_method(collection_type, options, :all)
+    end
+
+    def handled_list_method(collection_type, options = {}, method = :all)
```

Comment:
Right now it can be also :summary for non detailed list. I was thinking to allow it also relationships methods in the future, so instead of resources.all(:stack => stack) doing stack.resouces

But to your point, for fetching inventory, we will always get detailed non scoped list I guess. So I can delete it, I assume?